### DISCUSSION TITLE
Is adding freshwater the conceptually correct thing to do?

### DISCUSSION THREAD

**Discussion by gavinny** [NASA-GISS/freshwater-forcing-workshop#13] [Model implementation]
Neil?

**Comment by Recalculate** [comment]
From Andrew Meijers - don't make your professional github name based on your teenage gamer tag...The question as I understood it in the session was whether or not the cryosphere freshwater forcing should be included in the historical runs (and presumably piControl?), or kept in separate MIPs?  As a simple observational oceanographer, I feel it would be very valuable if this could be done.  The historical Southern Ocean (SO) is so poorly captured in the CMIP models, notably sea ice, temperature anomalies and vertical exchange, and at least some of these do appear to improve in the presence of added fw fluxes (eg Gavin's recent paper).  If we are leaving these fluxes out, then we are either getting the 'right' SO for the wrong reasons, or settling for a bad SO...which given its role in global heat/carbon uptake could have significant impacts.   Conceptually therefore it would seem to be the right thing to do; if we can.

I realise that adding fw makes things like getting a decent deep cell even harder, so perhaps the realistic answer is 'we want to but there are so many things to fix if we do, eg deep convection, dense water cascading etc, that it cannot realistically be done in time for CMIP7?  I'd be interested to know modeller's thoughts.

**Reply by xylar** [reply]
@Recalculate, thanks for your comment (and for identifying yourself)!

I also wonder if we're going to be able to come up with a freshwater forcing protocol that will be ready for the Fast Track. I think that's unlikely.  It feels to me that aiming for a MIP after Fast Track might be more realistic.  Presumably many modeling centers aren't going to be ready to adopt a new protocol.  This takes awhile to validate in each model and there may be resistance (as I found with my team working on the US Department of Energy's E3SM model) to adopting a new protocol but also little appetite for running new simulations in parallel to the planned ones to explore an alternative approach in time for Fast Track simulations.

**Reply by chrisdane** [reply]
Thanks for the joke and yes I agree @Recalculate: thousands of papers conclude over and over again that the physical processes horizontal and vertical mixing of active tracers are _the_ problem of (ocean) models. Working on adding additional freshwater (call it forcing or component, doesn't matter in my view @ Neil) of course is important and 'conceptually correct', but not the most important thing in my view.

**Reply by Recalculate** [reply]
@chrisdane I agree that mixing is critical, but we largely try to tune these as quasi free parameters to 'fix' the model differences to obs. If we do this in the SO to compensate for incorrect fw fluxes, we either wreck the rest of the ocean that uses the same parameterisations, or get the wrong SO values. I think the questions are: do we think we know the fw fluxes better than we know the 'correct' mixing parameters, and which is easier to fix? I'd argue without getting the fw fluxes right we'll forever be over tuning model mixing to compensate (in the SO at least).

**Reply by gavinny** [reply]
A few points that might help clarify some issues:
- I am personally most interested in the historical simulations since they are the ones being confronted by the observations. 
- Whether we need to do anything for the piControl depends on whether we think that the ice sheets were in quasi-equlibrium at that point. If yes, the anomalous FW is just zero in 1850. If we conclude otherwise, then we might need to think about using the last-millennium runs, or reconciling ourselves to either a forced drift in the control, or a bias, that might affect the spinup of the historical runs. 
- I don't think that anomalous freshwater can solve issues in the ocean model climatology - as Chris notes there is lots of scope for improving parameterizations (mixing, eddy-permitting/resolving resolution, atmospheric model improvements) that are likely more important. But it could still be ok to look at the trends and they are likely more affected bt the anomalous FW.

**Reply by mankoff** [reply]
I think it's better to assume things were not steady state. I'm not sure how to determine what state they were in though. Or at least if using a steady state assumption, make it clear this is a gross simplification.

**Reply by xylar** [reply]
I think most climate models are unprepared for the possibility that things were not in equilibrium in the their PI-control simulations.  Our methodology is to run very long simulations (1000 years or more) to quasi-steady state and to branch off of that simulation periodically to produce historical runs. We aren't prepared to insert FWF anomalies into such a simulation because it would inevitably violate the quasi-steady assumption.  I think it would be a big lift for this group to convince the climate modeling centers to change this approach.  So I would think we can't touch the PI simulations.

**Reply by nicojourdain** [reply]
Evolving from the assumption that the ice sheets are always in steady state to the assumption that they were in steady state in pre-industrial climate would be a major improvement and would not break the concept of climate model spin-up to some pre-industrial equilibrium. The different modelling groups could keep doing their spin-up the way they do, assuming steady ice sheet mass, save the final freshwater fluxes as background for the official piControl, then work with anomalies for the historical period and scenarios.

This is probably a reasonable assumption for Antarctica (e.g. [Braddock et al., 2022](https://doi.org/10.1038/s41561-022-00961-y)), but probably not so much for Greenland (see John Church's presentation), so there may be background trends (e.g. from the little ice age) to remove in order to calculate the anomaly from a supposedly steady pre-industrial climate...

**Reply by jlbamber** [reply]
@nicojourdain Also discussed in the obs. thread about GrIS stability. We can do a fairly good job at reconstructing GrIS trends from 1840-present and less so for the AIS. Frederiske 2020 manage to close the SLB with a ~zero AIS contribution for most of the 20th C. 
![TF table1](https://github.com/NASA-GISS/freshwater-forcing-workshop/assets/12460412/c6365076-68c1-4307-a911-b6c462d78c1b)

For the longer term then Kopp 2016 provides a sea level reconstruction for the Common Era with gradients of ~0.5 mm/yr. Of course you could have compensating trends for each ice sheet but other evidence (as you point to) suggests this is not the case so to first order assumption of steady state seems reasonable for pre industrial
![Kopp 2016](https://github.com/NASA-GISS/freshwater-forcing-workshop/assets/12460412/b1550070-a04e-44fb-a1b5-20b273df8a8e)

**Reply by therealpaulholland** [reply]
Most of the Antarctic ice mass loss is in the Amundsen Sea.  From sediment evidence, the ice configuration in that region was stable for approximately 10,000 years prior to the current retreat, which seems to have commenced in the 1940s.  e.g. 10.1038/nature20136.  So I think stable ice in preindustrial is fine.

**Reply by jlbamber** [reply]
@therealpaulholland . Yes agree. SLB closure gives similar conclusion for most of the 20th C but can't make same assumption for the GrIS.

**Reply by xylar** [reply]
I think whether it is correct or not, most ESMs are not prepared from a technological point of view to account for GIS being out of balance in the 1850s.  So approaches that can somehow compensate for this would, in my view, be necessary.

**Reply by gavinny** [reply]
So that I'm clear about what is being claimed, is this paraphrase correct: mid-19th C sea level rise was pretty close to zero. But is this just for the barostatic change or combined steric and barostatic change? Surely there was a small GIA component which is ~0.3 mm/yr sea level fall. However, GIS probably was more out of balance if anything - which is sufficient to cancel out the GIA?

**Reply by jlbamber** [reply]
@gavinny. Sorry misread your comment first time. What would be driving barystatic pre 1850? NH glaciers and ice caps (GIC), which are the largest volume compared to the total, were still growing in early 19th C and roughly reached their maximum extent between 1850-1900 depending on region. Could have internal variability driving AIS change but that's not well constrained.

For the Common Era, the rate fluctuated around ~ 0.5 mm/yr in RSL. The reconstruction by Bob is RSL not ASL, which is extremely difficult to do pre satellite. The GIA correction of 0.3 mm/yr is for ASL not RSL. Here's the GrIS reconstructed TMB for 1840-present. Obviously, pre 1950 it's not amazingly reliable but still useful IMHO
![180 year GrIS mb](https://github.com/NASA-GISS/freshwater-forcing-workshop/assets/12460412/e882efd6-3ef4-49a2-a861-f8f21b7e6673)

**Reply by jlbamber** [reply]
So, making a small contribution to SLR from 1840-1900 of ~0.05 mm/yr => 24 Gt/yr which is less than 1 mSv...

**Reply by jlbamber** [reply]
@gavinny in this paper we calc something we call the balance temperature, required for ASL to flucuate due to internal variability only and you will see that the value is pretty close to pre-industrial (if you discount the recent papr on sponges). Grinsted, A., J. Bamber, R. Bingham, S. Buzzard, I. Nias, K. Ng, and J. Weeks (2022), The Transient Sea Level response to external forcing in CMIP6 models, Earth's Future, n/a(n/a), e2022EF002696, doi:https://doi.org/10.1029/2022EF002696.

**Reply by bobkopp** [reply]
We addressed this in the Zoom chat, but for documentation purposes:

The centennial variability in Kopp et al 2016 dampened out in later reconstruction, c.f. 

- Walker, J. S., Kopp, R. E., Little, C. M., & Horton, B. P. (2022). Timing of emergence of modern rates of sea-level rise by 1863. Nature communications, 13(1), 966.

- Walker, J. S., Kopp, R. E., Shaw, T. A., Cahill, N., Khan, N. S., Barber, D. C., ... & Horton, B. P. (2021). Common Era sea-level budgets along the US Atlantic coast. Nature Communications, 12(1), 1841.

Key points from Walker et al. (2022):

- “Over the pre-industrial Common Era, the global component of reconstructed RSL exhibits fluctuating rates between −0.3 ± 0.2 and 0.2 ± 0.3 mm/yr from 0 to 1700 CE." [These are 60-year average rates]
- 0-1700 CE average rate of -0.08 ± 0.05 mm/yr (2s)

**Reply by jlbamber** [reply]
@xylar OK, useful to know. Assumption that FW anamoly for NH land ice is ~0 for 1850-1950 is not great. The flux is not huge and the uncertainties I quite large but post LIA both GrIS and Arctic GIC were losing mass, with lots of inter-annual varability of course. Magnitude is probably 2nd or 3rd order for the ocean but still there is a monotonic component. See, e.g. Kjeldsen, K. K., et al. (2015), Spatial and temporal distribution of mass loss from the Greenland Ice Sheet since AD 1900, Nature, 528(7582), 396-+, doi:10.1038/nature16183.

**Comment by swartn** [comment]
[Reformulated for clarity/brevity]

Are we ready to apply specified anomalous freshwater forcing in core policy relevant CMIP experiments, like historical, as opposed to testing these ideas and their many uncertainties more comprehensively in dedicated MIPs/experiments (e.g. NAhosMIP / SOFIAMIP / etc)?

All CMIP6 (and likely many CMIP7) models lack processes representing dynamic ice-ocean interaction, and we all agree the missing anomalous freshwater input is an important feedback on climate change. Adding anomalous freshwater as an additional "forcing" is arguably a good way to simply represent this missing interaction, given the large expense and difficulty in coupling an Ice Sheet Model. Indeed, this is the approach we have taken in the [SOFIA initiative](https://sofiamip.github.io/) and propose to extend into a [CMIP7 community MIP](https://airtable.com/appVPW6XAZfbOZjYM/shrtJ4jc08OEk7Vcq/tblAfxwzZTy4soluj/viwjIENyfUGTC5mfI/recABWC9hQjB7qbvD). 

However, I feel there are some possible complications with applying such forcing more generally in CMIP simulations outside of a dedicated MIP, which is focused on science not policy, and has clear goal and caveats. Some of these are:

- Climate change varies widely across models. In coupled ESM-ISMs, the icesheet derived anomalous freshwater input will vary in strength according to the model's climate change. In models without coupled ISMs, specifying a freshwater input timeseries (from observations or offline ISMs), will lead to some inconsistencies. For example, models with high and low climate sensitivity will experience the same anomalous meltwater input, and models with either warming or cooling CDW at the Antarctic shelf will experience the same freshwater input. Do we know the impacts of these inconsistencies, and how to interpret them? Would we need to, and could we explain this to the broader community using policy relevant experiments? (This arises in a sense because anomalous meltwater input is a feedback to climate change, as opposed to a purely external forcing).

-  The anomalous meltwater timeseries is I suspect far more uncertain than other core external forcings (as reflected in many of the observational talks). However, for practical application in DECK/historical, likely only a single timeseries could be specified (due to expense). Is our estimate input good enough to apply in policy relevant experiments? Maybe CMIP7 will treat historical uncertainty differently (Paul hinted at this), but quantifying this forcing uncertainty maybe easier/more appropriate in a dedicated MIP (could be in addition to doing it in historical).

- If applying anomalous freshwater as a forcing in CMIP7 historical, how are other experiments handled? CMIP6 had >100 different experiments, and CMIP7 will likely have more (most in community MIPs, but CMIP7 core will still contain dozens). It is not tractable to generate anomalous freshwater timeseries for all of these experiments. So how to decide which experiments to apply this additional forcing in and which to exclude? How does this affect interpretation, since many experiments refer back to DECK as a reference? For instance, in Detection and Attribution MIP, would an additional "meltwater-only" experiment be required to disentangle the effects? Would it be inconsistent having meltwater in the ALL forcing (historical) experiment, but not in GHG-only, or AER-only?

- Given the diversity of models, with varying representations of the relevant processes, is it plausible to design a consistent way to apply anomalous freshwater forcing, and on the timeline of CMIP7, is it possible for modelling centres to implement a new type of forcing correctly, and differentially across core policy relevant experiments? In SOFIA, we overcome this challenge by specifying a very simple design (surface uniform input), and then having higher tier experiments to test the sensitivity to how/where we input the freshwater. However, for CMIP7 core, how to design something "realistic" to apply across diverse models is not trivial, particularly when many modelling centres are actively working to explicitly include at least some of the relevant processes directly.

Whether in CMIP7 DECK or in a dedicated MIP, building greater community consensus on the best forcing timeseries/uncertainty and implementation into models is highly valuable, and this workshop is much appreciated. Thanks!

**Reply by nicojourdain** [reply]
Hi everyone,
conceptually, moving away from the assumption that the ice sheet mass is in steady state seems a good thing to me, even before having fully coupled ice sheet models that will capture the freshwater feedbacks. The Antarctic ice sheet mass loss over the last decades was due to a change in the ice sheet dynamics (driven by the ocean), not to a change in surface accumulation or runoff (e.g. [Rignot et al. 2019](https://doi.org/10.1073/pnas.1812883116), their Fig. 3), and it is the same with ice sheet model projections. In this respect using a freshwater flux anomaly equal to the net surface mass flux anomaly (precipitation minus surface runoff minus sublimation) as many climate models currently do seems to me more inconsistent than using an imperfect historical/scenario freshwater forcing.

**Reply by gavinny** [reply]
Hi Neil, 

- yes, ESM-ISM models will produce a range of FW inputs to the ocean, both in terms of absolute amounts and (very likely) in terms of FW fluxes per degree of global warming. There will also be sensitivity to the ocean circulation, ozone impacts on SAM, ISM complexity and parameterizations etc. But this is very analogous to models with interactive chemistry producing a range of CH4 lifetimes, or O3 responses. It's not inconsistent to run with CH4 concentrations though, nor would it be to run with observed FW amounts. 
-  I agree that the FW timeseries will be more uncertain than CO2 concentrations, say. But it might not be that much more uncertain than perhaps biomass burning.  It is probably sensible to explore the uncertainty in a MIP setup, but given that we know the sign and order of magnitude in the historical runs, assuming it's zero with zero uncertainty (as was done in CMIP6) seems sub-optimal
- dealing with the other experiments is worth discussing. I think for the future scenarios, using input from ISMIP6 or 7 is probably best, perhaps with some enhanced spread - this is something we can discuss this week. For DAMIP, there are several choices that could be made. How would it be split between hist-nat and hist-ant? (same issue with biomass burning too of course) given that we are not sure how to attribute the FW fluxes to specific drivers or long term responses perhaps to the deglaciation? The cleanest way would be to add a separate hist-fw experiment (which is what GISS will likely propose). It would of course be better to do hist-GHG etc in the ESM-ISM configurations, but we don't all have that ready to go yet. It could be possible to take early results from CESM or UKESM using their ISM configurations, or the ISMIP models., to derive a parameterization of the FW flux based on something (?) and use that in other models w/o ISMs, but I think that is further down the line. 
- For the historical runs, I think it's worth being more realistic with the implementation than one might be in an idealized setup. But there could be some simplifications that might help groups adopt this. The minimum you'd want (IMO - and we can discuss), is a spatial and depth profile drawn from observations or high resolution models (there is sufficient work being described in the talks that covers this, I think), regional variations in the FW fluxes could be derived, but it might be ok to impose a hemispherically uniform number for ease of implementation. There is a choice to be made with respect to phase for the component related to basal melt that enters the ocean at the coast. And there is a subtlety in what to do with the floating ice loss (should the flux account for the removal of seawater from the resolved ocean domain so that the impact on global sea level is correct?). But these are exactly the issues we should discuss in this workshop.

**Reply by swartn** [reply]
By inconsistent, what I mean is the externally specified FW flux would be different than a flux computed by processes responding to the local model climate. In the same way, specified O3 concentrations (from obs or another model) would differ from those computed by interactive chemistry. Depending on the context, I would submit that such differences induced by specification might be an acceptable approximation, or they may not. It depends on the question you are trying to answer, and the level of concern for internal consistency. I accept the premise that specifying an "inconsistent" FW flux is still likely a better approximation than zero flux in terms of reproducing historical.

**Reply by durack1** [reply]
@swartn @gavinny, I think you're both on track.

In my mind breaking it down to a multi-step process would be ideal:
1. Generate timeseries that attempts to quantify all freshwater components over a common period, e.g. 1901-present (or better 1850-present if possible to align with the existing ***CMIP:DECK:historical*** time coverage). Where my mind starts is the closed global sea-level budget in AR6 Ch9.6.1.2 with glaciers (~41%), GrIS (~25%), AIS (~4%), and land water (-~8%) - ala AR6 Table 9.5 ([here](https://www.ipcc.ch/report/ar6/wg1/chapter/chapter-9/#9.6.1.2)) and CCB 9.1, Fig 1 below - or quantify each of these components separately - do what is possible, with the full expectation that this will be an iteratively improved product with many varying versions as more constraints (and uncertainties!) are defined and applied
2. Create an anomalous freshwater forcing dataset that integrates these sources into a gridded time-evolving product (just lat/lon/time, or would ice sheets/land water require a vertical dimension?)
3. Engage with modelling groups to ascertain a protocol to convert the forcing dataset into a modelling-group usable product, so real freshwater volumes, salt-flux equivalents, are the terrestrial/glacier fluxes imposed through the land hydrology module, are they imposed independently as an additional flux (this will vary depending on the hydrology realism in the model, are all big freshwater outflows captured?) etc, and also how can these additional fluxes be generated to maintain consistency with the existing terrestrial water cycle operation/fluxes..
4. [Preferrably in a community MIP, ala SOFIAMIP] assess how big (in a multi-model context) the anomalous freshwater impact is, once the bugs in protocol have been nutted out by modelling groups (e.g., is salinity restoring, or other flux corrections required to maintain simulation integrity? And if yes, how can the impacts of these choices be accounted for to allow for the multi-model signal to be quantified?)
5. Come up with a protocol that makes sense across AOGCM, ESM, OGCM configurations that imposes (as much as possible) a consistent freshwater forcing across model configs
6. Propose this protocol as an amendment to the ***CMIP:DECK:historical*** (also need to consider esm-historical, and *piControl implications for spinup)
![Screenshot 2024-02-13 at 10 55 36 AM](https://github.com/NASA-GISS/freshwater-forcing-workshop/assets/3229632/fdb60a6c-74fb-4964-b089-188542236579)

**Reply by jlbamber** [reply]
Super useful. THanks. We actually have a pretty good SLB closure for the whole of the 20th C:
![TF fig 1](https://github.com/NASA-GISS/freshwater-forcing-workshop/assets/12460412/2b3523f5-c201-452c-a0d9-0da66a62bd6d)

**Reply by swartn** [reply]
Sounds good @durack1, @gavinny 

To ensure we are aligning not duplicating efforts, it would to consider how SOFIAMIP might fit into this. For example, it could be the place to explore uncertainties in historical / future forcing or other assumptions. The is no specified design yet, so it could work off whatever forcing consensus is derived here (what we have done in SOFIA initiative so far is separate). I think that would be more useful than if these activities are unaligned. Certainly, having a consensus estimate of the historical forcing will be very useful.

I guess it remains to be seen whether FW would be accepted formally into CMIP core runs, but either way integration would be good.

**Reply by durack1** [reply]
@jlbamber nice nice nice! Ok so maybe next, how regionally can these contributions be defined? For GrIS/AIS do we have regional contributions (so ocean connections - lat/lon/depth/time?) and with glaciers can these be mapped to their sources, which could maybe then be mapped into model hydrology grids? (the can of worms here would be maintaining consistency with the model's own terrestrial hydrology/water cycle while pouring a huge amount of freshwater downstream from glaciers.. - @swartn's point about process consistency)

As an FYI for me, does the glacier contributions jive with the ongoing [GlacierMIP](https://climate-cryosphere.org/glaciermip/) (and just because [GlacierMIP3](https://github.com/GlacierMIP/GlacierMIP3)) activities?

**Reply by durack1** [reply]
> I guess it remains to be seen whether FW would be accepted formally into CMIP core runs, but either way integration would be good.

My two cents. There is a (ridiculously, I think this word is warranted) ambitious timeline for AR7 (which is yet to be finalized), and consequently CMIP7 that will be putting pressure everywhere (forcing, modeling groups, infrastructure, etc). I advocate that this community/activity works on a timeline that is science (not IPCC or CMIP-phase) driven, work out the kinks and then engage with modelling groups when there is a tested framework to implement.

We've been hoping a "continuous DECK" to be realized since CMIP6, with evolving forcings (somewhat continuously, "operational" is the aspiration), which means that a 2024 historical (and definitely a CMIP6 historical) and 2026 historical simulation, etc may be quite different, as the forcings (and science driving this) have evolved.

**Reply by jlbamber** [reply]
@durack1 No single answer to first q. as it depends on the time period. For satellite era it will be as good as you need. Prior to that regional resolution degrades rapidly. For AIS, pre 1979, really we can only say at the whole ice sheet scale but then the anomaly is small (probably) for early 20th C. For glaciers, we just discussed that briefly and I guess we'd generate a mass anomaly that would map to the scale of the hydrology or land surface scheme grids but these are the kind of detail that are important to fix early on.

**Reply by durack1** [reply]
@jlbamber is your SLB figure (5 comments up) with all the goodies 1900 to ~2020 published somewhere?? This is a very, very nice figure that I would like to educate myself all about!

**Reply by ben-davison** [reply]
@durack1 it's from Frederikse et al. (2020): https://www.nature.com/articles/s41586-020-2591-3

**Reply by jlbamber** [reply]
@durack1 So the paper by Frederikse is pretty important as it "solves" the long standing Munk sea level enigma re. the early 20th accel in sea level and the sources. It's also a really nice analysis of uncertainties in 20th C SLB. There's been a lot of work on the SLB during the altimetry era and this paper is pretty comprehensive https://essd.copernicus.org/articles/10/1551/2018/essd-10-1551-2018.html

**Comment by mankoff** [comment]
Historical balance for Greenland is addressed by Box and Colgan (2013), and that is then somewhat improved and incorporated into Mankoff et al. 2021 (http://doi.org/10.5194/essd-13-5001-2021). For the record, I don't love the pre-1980 data and there are large uncertainties. 

:warning: This product only provides net SMB and discharge, which is not sufficient to provide total freshwater. For example, a year with net 0 SMB could have high snowfall and high meltwater runoff, or low SMB inputs (snowfall) and low SMB outputs (runoff). For a discussion of the difference between mass balance and freshwater, see #6 .

**Comment by therealpaulholland** [comment]
So, if I understand it, currently most models assume steady state ice, implying that the historical and future surface mass balance trends over the ice sheets lead to an anomalous freshwater flux into the ocean.  Can anyone point me towards a study that looks at the numbers for these, in the CMIP models?  Maybe they are of the correct magnitude?

**Reply by gavinny** [reply]
Purich and England (2023) might be useful (as mentioned in Matt England's talk) https://doi.org/10.1175/JCLI-D-22-0457.1 . The increase in FW flux that balances the increased snowfall is much smaller than the inferred change in flux:
<img width="702" alt="image" src="https://github.com/NASA-GISS/freshwater-forcing-workshop/assets/24462428/e2f61263-c33a-4c70-af2b-51efda680907">

**Reply by therealpaulholland** [reply]
Thanks.  So for the historical period the models may already be correct?  but for the very high end estimates under SSP5 for the future it looks like they are not

**Reply by hthewitt** [reply]
When I look at the figure from Purich and England (2023) and listen to some of the discussion, I think that to be sure that something is produced that is relevant to all models, we do need absolute clarity on what the ESMs are doing with freshwater (eg, it's an out of date version now but we did this for HadGEM3 in Hewitt et al, 2010) and that the models are actually conserving freshwater

**Reply by therealpaulholland** [reply]
BTW the values shown in purple above are pretty high numbers, I think.  In the UKESM1.0-ice (with dynamic ice sheets) we find a small net sea level fall from Antarctica by 2100 under RCP8.5, implying that precipitation increase is about the same as dynamic mass loss during that period.  In a steady state climate model that would have meant the FW flux increase in RCP8.5 was about right.  But there are large uncertainties about all of this and we can discuss in later sessions.

**Reply by swartn** [reply]
@therealpaulholland 

Here, for CanESM5 at least, is the Antarctic runoff flux, smoothed with a 10 year running mean. It increases by about 200 Gt/yr over historical (quite a but less than the increase of 600 Gt/yr we estimated from obs. papers [here](https://egusphere.copernicus.org/preprints/2023/egusphere-2023-198/egusphere-2023-198.pdf)). This is what I would expect, since this change only reflects changes in SMB. The model is missing any ocean-ice interactions, which contribute to most of that observed figure.

Over 1981 to 2000, the CanESM5 flux is 2000 Gt/yr, consistent with the bar shown above in Purich et al.

![image](https://github.com/NASA-GISS/freshwater-forcing-workshop/assets/6588913/94f2d060-97f7-4b62-8f1e-2ec9ed53a17d)

**Reply by durack1** [reply]
@swartn naive question, what is driving the ~50-year periodicity, ~1900, ~1950, ~2000 peaks. And what is going on in 1875 and ~1900?? It's a period of very weak positive forcing, with the big volcanic events Krakatoa 1883, Mount Tarawera 1886, Mount Bandai 1888 kicking OHC change in the other direction...

**Reply by therealpaulholland** [reply]
@swartn great plot, thanks!  I wonder how much this varies between models, for historical and future periods.  The magnitude is a bit awkward, isn't it?  It's big enough that it is not clear it can be ignored, but small enough that there is a material problem with the FW magnitude.  Where did the workshop get to with handling this signal?  I guess there are 3 options:

1) leave this in there, and add obs FW timeseries over the top
2) subtract this timeseries in its entirety, and replace with obs FW timeseries
3) subtract a low-order curve fit to this timeseries, and replace with a low-order curve fit to the obs FW timeseries.

As discussed my vague instinct is to do option 3. i.e. I would probably keep a lot of the variability from the above.  But that raises the question, how would we know how much variability to keep?  Maybe it is simpler to just follow options 1 or 3.  I really don't know.

**Reply by swartn** [reply]
@durack1 -I'm not sure what is driving the periodicity. Mechanistically I believe it must be related to P-E changes (could be surface melt, but probably unlikely). We do have fairly large oscillations in e.g. Drake Passage transport, which could be associated with a broader SH climate response. Something to look into.

@therealpaulholland - I agree, the magnitude is a bit awkward. @gavinny has a proposal for what to do over in #5, which is a bit like you 2, but only subtracts the anomaly from piControl. Any option has challenges though.

For what it's worth, in SOFIA, we essentially do your 1. The logic is, most of the anomalous FW comes from enhanced ice shelf melt and calving, which is driven by ocean warming, and not represented in the models. Especially when going into the future, the SMB/runoff increases are small compared to estimate of anomalous FW increase (as shown in Purich bars).

**Reply by therealpaulholland** [reply]
Maybe the oscillations are Weddell Polynyas periodically opening up.  Thanks for pointing me to #5, where I found the answer but then followed this up with a new slightly tangential point.

**Reply by Recalculate** [reply]
(Andrew Meijers here) @swartn I have a student, Jon Rosser, looking at exactly the driver of oscillations in CanESM5.  In the piControl the period is closer to 100 years.  Associated with this is a mode of heat distribution in the SO south of 40, oscillating around 60S, driven by the deep cell and the presence/absence of coordinated dense water formation in the Weddell/Ross seas.  The 100 year timescale (and presumably the synchronicity of both gyres) appears to be associated with the transit time of dense water from the formation sites to the ACC, and the subsequent return flow of slightly lighter CDW.  It seems to be something like dense water production in the southern gyres -> increased AABW export/southern intensification/ACC acceleration -> increase in gyre size and southward GM eddy flux, entraining relatively warmer waters into gyres -> warmer return flow shutting down dense water formation -> cessation of warmer inflow -> convection starts again.      Jon has noted similar behaviour in a couple of other CMIP6 models, as well as a different mechanism for  oscillations driven in the GFDL model that is more of a deep heat accumulation -> tipping point -> wild convection/ice loss/heat loss -> re stratification.

We've not looked at the precipitation associated with these runs, but there is a significant surface warming/sea ice loss associated with the cycle, so I'd not be terribly surprised if that impacted evap/precip regionally.  Interestingly surface fluxes over the gyres seems to respond to the oscillation, rather than drive it, and there isn't any meaningful signal in the freshwater fluxes/winds/atmospheric modes that might hit at an external driver.  Likewise if you look at hovemollers of the ocean heat content, the SO signal propagates north, but only weakly once you look past 40S, so it doesn't really appear to be linked to wider AMOC variability either.  It seems to be very much a SO internal oscillation.  

He has looked at forced runs of both the CanESM5 and HadGEM3 (which has somewhat similar behaviour after a point in its run), and interestingly noted that under most forms of increased climate forcing the oscillations are very quickly damped out, presumably by increased stratification.  However, in runs with 1/2 co2 forcing, the oscillations reappear, but with a 50 year cycle!   We're still wondering about what that is, and any suggestions would be welcome.  In any event, it seems that these models exist in something of a 'goldilocks zone' where it is just right for sustained oscillations of this nature, but moving the mean state much seems to kill them (or change their frequency).

**Reply by swartn** [reply]
@Recalculate -yes I have spoken to Jon a few times and always interested to hear his progress - I agree that cycle is predominantly 100 y. We also see in our SOFIA runs, with only FW forcing, that the oscillations completely disappear. The part about shifting to a 50yr cycle under 1/2 CO2 is interesting.

**Reply by swartn** [reply]
I wonder if an expanded analysis like this might fit into the workshop paper, or perhaps merit a paper. I.e. examining the evolution and drivers of Antarctic "runoff" in CMIP6, as a basis for understanding how prescribed FW fluxes would differ from the status quo.

**Reply by torgemartin** [reply]
Just knowing the spread in runoff magnitudes from Antarctica for CMIP6 models would be an insightful basis for the discussion here, I guess. But in principle one would also need to have information on how precipitation (liquid and solid) is treated individually and routed to the coast, how the ice sheet is represented (thin snow layer, near-infinite snow pile, ...), is there any time delay included (seasonal effects, e.g. precip timing, instant/delayed runoff, delayed iceberg "calving", ...), to actually know what these models are doing and see the potential wealth of individual solutions.

**Reply by swartn** [reply]
Agreed @torgemartin . The survey @gavinny did of modelling centres provides a lot of insight into what was done in this respect.

**Comment by mankoff** [comment]
@williamcolgan  (not part of this workshop so far?) may have thoughts on historic Greenlandic mass balance (and mass balance partitioning).

**Reply by WilliamColgan** [reply]
Thanks for the H/T Ken.

Hi Folks -- Yes, I did once spend a lot of time looking into the influence of assumed pre-satellite era GrIS balance flux on our understanding of contemporary GrIS mass loss partition (i.e. during the satellite period). Long story short, previous studies have assumed a variety of balance fluxes from the GrIS. Generally, the lower balance flux you assume, the higher the contemporary mass loss you attribute to ice dynamics. 

So, in terms of this thread, I agree with @jlbamber -- I would encourage folks to assume a negative mass balance baseline for the GrIS during the 1850-1950 period, largely based on Kjeldsen2015. Imposing an equilibrium assumption during the historical period will otherwise bias both the iceberg calving and SMB contributions differenced from the historical flux. 

I attach some relevant figures here, and can share the triple-rejected, and now quite dead, manuscript if anyone wants. 

![reboot_Fig1](https://github.com/NASA-GISS/freshwater-forcing-workshop/assets/88824039/3abeedb9-9014-46a7-accc-e07ccb15ddba)
![reboot_Fig2](https://github.com/NASA-GISS/freshwater-forcing-workshop/assets/88824039/52201b47-b46e-4505-bef7-be6f4fbd8e8f)
![reboot_Tab1](https://github.com/NASA-GISS/freshwater-forcing-workshop/assets/88824039/64588a7c-7e2f-4e13-8470-60fba0f66d6a)

Abstract: 
Accurately partitioning past- and present-day ice-sheet mass loss into meltwater runoff and iceberg discharge fractions is critical for accurately projecting future mass loss partition. The sea-level rise contribution of iceberg discharge (or meltwater runoff) is the departure of this flux from a theoretical balance flux. Here, we develop a novel framework to estimate this balance flux, which avoids the common assumption of recent ice-sheet equilibrium. Our balance flux of 422±57 Gt/yr resides within the ~130 Gt/yr range of previous balance flux estimates. We difference this balance flux from an ensemble of iceberg discharge estimates to infer that iceberg discharge was responsible for 52±24 % (or 5.4±0.4 mmSLE) of ice-sheet mass loss over the 2000-2018 period. We find that ice dynamic mass loss from Southeast Greenland is disproportionately low in comparison to that from Northwest Greenland. We speculate that longer term ice dynamic thickening may be buffering the recent ice-dynamic sea-level rise contribution from South Greenland.
